### PR TITLE
Fix eagerly finding fdfind/fd

### PR DESCRIPTION
**Pull request by jcf**
Previously this would call `(executable-find "fdfind" t)`. The function
signature of `executable-find` is as follows:

    (executable-find COMMAND)

### CODE REVIEW

File: core/core-projects.el [hunk 1]
```diff
@@ -28,7 +28,8 @@ Emacs.")
   (setq projectile-cache-file (concat doom-cache-dir "projectile.cache")
         projectile-auto-discover nil
         projectile-enable-caching doom-interactive-p
-        projectile-globally-ignored-files '(".DS_Store" "Icon" "TAGS")
+        projectile-globally-ignored-files '(".DS_Store" "Icon
+" "TAGS")
```
Comment (jcf) [inline core/core-projects.el:32]: Sorry, this is an unrelated change that the GitHub UI added.

File: core/core-projects.el [hunk 2]
```diff
@@ -154,7 +155,7 @@ And if it's a function, evaluate it."
           ;; `projectile-globally-ignored-directories' are respected.
           (lambda (_)
             (concat (format "%s . -0 -H -E .git --color=never --type file --type symlink --follow"
-                            (or (cl-find-if (doom-rpartial #'executable-find t)
+                            (or (cl-find-if (doom-rpartial #'executable-find)
```
Comment (jcf) [inline core/core-projects.el:158]: This is the relevant change. :point_up:

**Comment by hlissner** [comment]
A different solution was needed. This should be fixed as of 00ebbaf.

**Comment by jcf** [comment]
Thank you, @hlissner! :bow:
